app/controllers/api/v1/users_controller.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module API
+  module V1
+    class UsersController < API::V1::ApplicationController
+      include API::V1::ErrorHandlerConcern
+
+      skip_before_action :doorkeeper_authorize!, only: :create
+
+      before_action :ensure_valid_client, only: :create
+
+      def create
+        user = User.new(create_params.except(:client_id, :client_secret))
+
+        render_errors ActiveModel::ErrorsSerializer.new(user.errors).serializable_hash and return unless user.save
+
+        render json: UserTokenSerializer.new(user, { params: { client_id: @client_app.id } }).serializable_hash,
```

Comment:
@malparty I guess it should work without calling `serializable_hash` as well? Can you try it here?

Reply:
Tried and YES, it works inside as render section. Though it does not work in other places (like UserTokenSerializer). 

Good to know! :) 

And... fixed! ✅